src/flamenco/types/fd_types.json

```diff
@@ -457,6 +486,46 @@
       ],
       "comment": "https://github.com/solana-labs/solana/blob/88aeaa82a856fc807234e7da0b31b89f2dc0e091/runtime/src/bank.rs#L967"
     },
+    {
+      "name": "serializable_versioned_bank",
```

Comment:
(note for future): as the only difference between `serializable_versioned_bank` and `deserializable_versioned_bank` is the usage of `solana_account` vs `solana_vote_account` in the stakes cache, we can probably make both use `solana_account` and just read the extra `last_timestamp_ts`/`/last_timestamp_slot` fields from the account state in Funk when we need them.  It appears that we don't use the `node_pubkey` field at all, and we only use the timestamp fields in one place (`recover_clock`). 

This would avoid us having to have two versions of this `versioned_bank` data structure.

Just a thought 😄

Reply:
Yes I agree, didn't want to muck it up too much in this PR. We should always just be using the serializable one... I'll leave a todo